internal/command/machine/run.go

```diff
@@ -716,6 +726,16 @@ func determineMachineConfig(ctx context.Context, initialMachineConf api.MachineC
 		machineConf.Init.Entrypoint = splitted
 	}
 
+	// default restart policy to always unless otherwise specified
+	switch flag.GetString(ctx, "restart") {
+	case "no":
+		machineConf.Restart.Policy = api.MachineRestartPolicyNo
+	case "on-fail":
+		machineConf.Restart.Policy = api.MachineRestartPolicyOnFailure
+	default:
+		machineConf.Restart.Policy = api.MachineRestartPolicyAlways
```

Comment:
wdyt if it fails in case the value is not one of the allowed? been loose can lead to gotchas when someone mistype any of the other values.

Reply:
Good idea, updated!